customers/models.py

```diff
@@ -19,6 +19,24 @@
 User = get_user_model()
 
 
+class CustomerProfileManager(models.Manager):
+    def get_queryset(self):
+        return (
+            super()
+            .get_queryset()
+            .annotate(
+                customer_group=Case(
+                    When(
+                        organization__isnull=False,
+                        then="organization__organization_type",
+                    ),
+                    default=Value("private"),
+                    output_field=models.CharField(),
```

Comment:
Wouldn't this be a bit redundant having the same value as with `organization type`?Like, you could assume that if `organization == null` customer will always be private.

Reply:
that is exactly what the client is doing right now. but since we wanted to have a "customerGroup" filter, I thought it would be easiest if the value is annotated directly on the profile. now the client can see `CustomerGroup` enum values and use them to fitler `berthProfiles`

Reply:
🤔 I mean it makes sense and if it's gonna make the client's work easier, then let's go for it. 👍